ibm_catalog.json

```diff
@@ -80,90 +81,133 @@
           },
           "iam_permissions": [
             {
-              "service_name": "containers-kubernetes",
               "role_crns": [
-                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Viewer"
+              ],
+              "service_name": "Resource group only",
+              "notes":"Viewer access is required in the resource group you want to provision in."
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Administrator"
+              ],
+              "service_name": "All Account Management services",
+              "notes": "[Optional] Required for consuming Account Configuration deployable architecture which creates resource group."
```

Comment:
I suggest simplifying this note, as most consumers may not realize there's an additional DA involved.

[Optional] Only required if the option to create new resource groups is enabled (Account Configuration setting).